cypress/integration/spec.js

```diff
@@ -76,31 +88,90 @@ describe('Chart Explorer', function() {
     ).click()
     cy.get('[data-test=rm-encoding]').click()
 
-    // check for editor existence
-    cy.get('#configure-tabs-tab-editor').click()
-    cy.get('.monaco-editor')
-    cy.get('#configure-tabs-tab-builder').click()
-
     // check that titles work
     cy.get('[data-test=chart-title]').type('test title')
     cy.get('svg .role-title').contains('test title')
 
     // check that chart exists
     cy.get('[data-test=vega-embed]')
-
     cy.screenshot('page with viz')
 
-    // check download options
+    cy.get('[data-test=license-open]').click()
```

Comment:
I think we can create utility selector functions like we have in `ui/cypress-tests` since the `[data-test=...]` should be abstracted. Maybe something like `utils/common`:

```
export function getDataTestAttrSelector(element, selectorName, modifier = '') {
  return `${element}[data-test="${selectorName}"]${modifier}`
}

export function getElementByDataTestAttr(element, selectorName, modifier = '') {
  return cy.get(getDataTestAttrSelector(element, selectorName, modifier))
}
```

Reply:
+1 to this idea. @DanialBetres it seems like the `data-test` attributes are serving the same purpose as `data-dw` in `core-web-app`. I'd even go as far to say we should convert them to `data-dw` attributes. based on your experience working in `chart-builder`, can you think of a reason why we wouldn't want to do this?

Reply:
@tkhanna42 @chorn21  Sounds good, I'll convert them to `data-dw` attributes.

But I'm curious why `[data-test=...]` should be abstracted? It seems like usage of it is simple and non-verbose (at least in the cases i'm using them in)